### ISSUE TITLE
Need to use only 75 percent of threshold value

### ISSUE THREAD

**Issue by 104H** [104H/dyadicdecisionmaking#45]
> Throughout each run, the contrast of the grating (i.e., the signal strength)
> was fixed at each subject’s 75% correct detection threshold level
>
> _deGee 2014_

**Comment by nataliasc** [comment]
This is already defined in the staircase procedure

**Comment by zora-no** [comment]
Lucas and Artur are still discussing this. I'll update here

**Comment by zora-no** [comment]
They are pretty sure that the output threshold is already a 75% threshold, but Artur said that we should confirm this by 1. doing a titration and 2. doing many trials with this threshold and 3. checking how many times we were correct.

**Comment by nataliasc** [comment]
If it helps, I found [this reference](https://www.sciencedirect.com/science/article/pii/S0042698997003404) from PsychoPy forums, and according to it, the `pThreshold` value we have (.63) should converge to 77.85% correct answers

**Comment by 104H** [comment]
**Mentioning some relevant parts from the paper references above.**

> The length of a staircase is usually specified as a fixed number of reversals (see Table 1, column E), and the staircase finishes after however many trials are required to reach that number of reversals. Lengths for the staircases in this study ranged from a few to a few thousand reversals.

> Formally, asymptotic convergence of a staircase is the limit of the average of the reversal values (or any other final measure obtained from the staircase trials, for that matter) as the number of reversals increases, i.e. the limit of Eq. (9)or Eq. (10)as N tends to infinity. 

> Two hundred independent staircases requiring 10 000 reversals each were simulated in each condition.

**Comment by nataliasc** [comment]
I think our `startVal` and `startValSd` are not too good. Firstly, we already know from all the pre-pilot runs that everyone can see the grating even when its contrast is as low as 0.1. So the start value (initial guess) of 0.5 is way too high.
And also, there is this part in the [documentation](https://www.psychopy.org/api/data.html#psychopy.data.QuestHandler) that says
>**startValSd:**
> Standard deviation of your starting guess threshold. Be generous with the sd as QUEST will have trouble finding the true threshold if it’s more than one sd from your initial guess.

The true threshold in the past experiments has always been further than one sd away from 0.5, so this could be the reason why our titration script is now always finding threshold that is too high for our purposes

**Comment by 104H** [comment]
@frbosse and I tried 0.051 mean and 0.2 sd on Friday and we had a combined accuracy of 0.2 on 2 blocks of 80 trials.

**Comment by nataliasc** [comment]
Do you mean you tried an initial guess of 0.051?

**Comment by 104H** [comment]
Yes, the startVal and the startValSd arguments, as provided to the titration script on line 171 and 172 respectively.

**Comment by nataliasc** [comment]
I found [this demo](http://psychtoolbox.org/docs/QuestDemo) today which is easier to understand than PsychoPy's docs, and maybe some information in it will turn out helpful

**Comment by nataliasc** [comment]
Also this old issue https://github.com/psychopy/psychopy/issues/1005 is relevant, and it is still open
